### ISSUE TITLE
Library Color and row

### ISSUE THREAD

**Issue by kek001** [mixxxdj/mixxx#11141]
### Feature Description

Is it possible have a feature which turns off full row color in library, if using  a color. so the color would be in color column only and not full row. It makes library little bit harder to read now.

**Comment by ronso0** [comment]
Please provide an example screenshot.
Maybe it's sufficient to adjust the font color, or make it switch between bright and dark depending on the background.

**Comment by kek001** [comment]
I would like to disable coloring full row, and keep it in color column only.

![MixLibColor](https://user-images.githubusercontent.com/79098298/208876090-c0311d71-beff-4cca-a574-2519703843f0.png)

**Comment by ronso0** [comment]
Sure, this is possible.

Meanwhile you could try another font. IMO the narrow font you selected is hard to read even with regular contrast.

**Comment by kek001** [comment]
> Sure, this is possible.
> 
> Meanwhile you could try another font. IMO the narrow font you selected is hard to read even with regular contrast.

yes i could use better font too, i edited later night mixxx default skin.

**Comment by palaran** [comment]
Would be useful if there could be slight outline of the colour blocks/line between rows so it is easier to separate which colour applies to which track when looking along the rows.

**Comment by daschuer** [comment]
Can you edit a mock up using Gimp or such?

**Comment by palaran** [comment]
Ok, very rough but hopefully gives you the idea. A bit like putting borders round a table in excel.
![Colours](https://user-images.githubusercontent.com/91451851/214017344-8758bf85-0b16-4501-b684-98bb015d861c.jpg)

**Comment by ghztomash** [comment]
I would also like to see this feature implemented

**Comment by ronso0** [comment]
This can be implemented like the "Grey out played tracks" option.
The rows would not be painted and the color delegate would still work.

Easy task, just tedious.
